src/ImageHeaderScrollView.js

```diff
@@ -2,9 +2,12 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Animated, ScrollView, StyleSheet, View, Image, Dimensions } from 'react-native';
+import OverflowView from 'react-native-view-overflow';
 import type { ViewProps } from 'ViewPropTypes';
 import type { FlatList, SectionList, ListView } from 'react-native';
 
+const AnimatedOverflow = Animated.createAnimatedComponent(OverflowView);
```

Comment:
This component decrease the animation perf a lot. I removed it. What was the usecase ?

Reply:
I had content that I needed to overflow on Android. Android can overflow now since a new release of RN came out, so this isn't necessary anymore